api/v3/GroupContact.php

```diff
@@ -256,43 +200,3 @@ function _civicrm_api3_group_contact_common($params, $op = 'Added') {
   $dao = NULL;
   return civicrm_api3_create_success(1, $params, 'GroupContact', 'create', $dao, $extraReturnValues);
 }
-
-/**
- * Update group contact status.
- *
- * @deprecated - this should be part of create but need to know we aren't missing something
- *
- * @param array $params
- *
- * @return bool
- * @throws \CRM_Core_Exception
- */
-function civicrm_api3_group_contact_update_status($params) {
-
-  civicrm_api3_verify_mandatory($params, NULL, ['contact_id', 'group_id']);
-
-  CRM_Contact_BAO_GroupContact::addContactsToGroup(
-    [$params['contact_id']],
-    $params['group_id'],
-    $params['method'] ?? 'API',
-    'Added',
-    $params['tracking'] ?? NULL
-  );
-
-  return TRUE;
-}
-
-/**
- * Deprecated function notices.
- *
- * @deprecated api notice
- * @return array
- *   Array of deprecated actions
- */
-function _civicrm_api3_group_contact_deprecation() {
-  return [
-    'delete' => 'GroupContact "delete" action is deprecated in favor of "create".',
```

Comment:
@KarinG looks like there is a drop-in replacement (use `create` instead of `delete`). How hard would it be to patch the module with that switch?

But given the usage-in-the-wild, we should probably also revert at least that one bit of this PR.